app/controllers/discourse_ai/admin/ai_personas_controller.rb

```diff
@@ -32,6 +36,11 @@ def create
         end
       end
 
+      def create_user
+        user = @ai_persona.create_user!
```

Comment:
I wonder if we should make these into actual non-human (negative ID) users? I know we have no system yet for "reserving" these in core so it does make things trickier. Maybe we could start at `-10_000` for these AI bot users and count down from there, using a postgres sequence could make this easier:

```
CREATE SEQUENCE ai_bot_user_ids AS integer INCREMENT BY -1 START WITH -10001;
SELECT nextval('ai_bot_user_ids');
=> -10001
```

Maybe this is too complex, but I am just thinking it may get confusing to have regular users floating around that are _really_ bot users.

Reply:
yeah something about "flagging bots" is a bit of a dark science ... I can at a minimum pick a negative id out nowhere ... will do that for now

Reply:
I moved to using negative numbers in a reserved range, should be much better now... not ideal given there is no range registration but better for sure.